tests/lca/imagebasedinstall/mgmt/deploy/tests/e2e-deploy-test.go

```diff
@@ -50,110 +58,182 @@ var _ = Describe(
 			ibiImageSetName = strings.Join(splitOCPVersion[:2], ".")
 		})
 
-		It("through IBI operator is successful in a connected environment", reportxml.ID("76641"), func() {
-
-			tsparams.ReporterNamespacesToDump[MGMTConfig.Cluster.Info.ClusterName] = "spoke namespace"
-
-			By("Create namespace for IBI installation")
-			_, err := namespace.NewBuilder(APIClient, MGMTConfig.Cluster.Info.ClusterName).Create()
-			Expect(err).NotTo(HaveOccurred(), "error creating namespace")
-
-			By("Get pull secret from hub cluster")
-			spokePullSecret, err := secret.Pull(APIClient, "pull-secret", "openshift-config")
-			Expect(err).NotTo(HaveOccurred(), "error getting pull-secret from hub cluster")
-
-			By("Create pull secret for spoke cluster")
-			spokePullSecret.Definition.Name = MGMTConfig.Cluster.Info.ClusterName
-			spokePullSecret.Definition.Namespace = MGMTConfig.Cluster.Info.ClusterName
-			spokePullSecret.Definition.ResourceVersion = ""
-			_, err = spokePullSecret.Create()
-			Expect(err).NotTo(HaveOccurred(), "error creating spoke pull-secret")
-
-			for host, info := range MGMTConfig.Cluster.Info.Hosts {
-				By("Create baremetalhost secret for " + host)
-				_, err = secret.NewBuilder(
-					APIClient, host, MGMTConfig.Cluster.Info.ClusterName, v1.SecretTypeOpaque).WithData(map[string][]byte{
-					"username": []byte(info.BMC.User),
-					"password": []byte(info.BMC.Password),
-				}).Create()
-				Expect(err).NotTo(HaveOccurred(), "error creating bmh secret")
-
-				By("Create baremetalhost for " + host)
-				hostBMH := bmh.NewBuilder(
-					APIClient, host, MGMTConfig.Cluster.Info.ClusterName, info.BMC.URLv4, host, info.BMC.MACAddress, "UEFI")
-				hostBMH.Definition.Spec.AutomatedCleaningMode = "disabled"
-
-				_, err = hostBMH.Create()
-				Expect(err).NotTo(HaveOccurred(), "error creating baremetalhost")
-			}
+		It("through IBI operator is successful in a connected environment with static networking",
```

Comment:
do we want to specify the IP family?
it's expected to have ipv4 for connected at the moment, but it could be either.

Reply:
I need to actually add the ability for us to check if a cluster is connected and if it is IPv4